data/culture/fil.lua

```diff
@@ -0,0 +1,366 @@
+
+-- Filipino names
+
+-- References
+-- Wikipedia: Filipino names - https://en.wikipedia.org/wiki/Filipino_name
+-- https://github.com/Ninble/name-census-top-100 (GPL-3.0)
+
+local CultureName = require './common'
+
+local male = {
+    "Aaron",
+    "Adrian",
+    "Albert",
+    "Alejandro",
+    "Alex",
+    "Allan",
+    "Alvin",
+    "Andrew",
+    "Angel",
+    "Angelo",
+    "Anthony",
+    "Antonio",
+    "Brian",
+    "Bryan",
+    "Carl",
+    "Carlo",
+    "Carlos",
+    "Charles",
+    "Chris",
+    "Christian",
+    "Christopher",
+    "Cj",
+    "Dan",
+    "Daniel",
+    "Dave",
+    "David",
+    "Dennis",
+    "Don",
+    "Edward",
+    "Efren",
+    "Eric",
+    "Francis",
+    "Gabriel",
+    "Ian",
+    "Ivan",
+    "Jake",
+    "James",
+    "Jan",
+    "Jason",
+    "Javier",
+    "Jay",
+    "Jayson",
+    "Jean",
+    "Jeff",
+    "Jeffrey",
+    "Jen",
+    "Jerome",
+    "Jesús",
+    "Joan",
+    "Joel",
+    "Joey",
+    "John",
+    "John-Paul",
+    "Jonathan",
+    "José",
+    "Joseph",
+    "Josh",
+    "Joshua",
+    "Juan",
+    "Julius",
+    "Justin",
+    "Karl",
+    "Ken",
+    "Kenneth",
+    "Kevin",
+    "King",
+    "Kyle",
+    "Leo",
+    "Luis",
+    "Manuel",
+    "Marc",
+    "Marco",
+    "Mark",
+    "Mark-Anthony",
+    "Martín",
+    "Marvin",
+    "Michael",
+    "Miguel",
+    "Mike",
+    "Neil",
+    "Pablo",
+    "Paolo",
+    "Patrick",
+    "Paul",
+    "Prince",
+    "Ralph",
+    "Raymond",
+    "Richard",
+    "Robert",
+    "Ron",
+    "Ronald",
+    "Ryan",
+    "Sam",
+    "Shane",
+    "Toni",
+    "Victor",
+    "Vince",
+    "Vincent",
+}
+
+local female = {
+    "Abby",
+    "Aira",
+    "Alyssa",
+    "Ana",
+    "Andrea",
+    "Angela",
+    "Angelica",
+    "Ann",
+    "Anna",
+    "Anne",
+    "April",
+    "Arlene",
+    "Ashley",
+    "Bea",
+    "Bianca",
+    "Camille",
+    "Carla",
+    "Carmen",
+    "Catherine",
+    "Charlene",
+    "Charmaine",
+    "Cherry",
+    "Christine",
+    "Claire",
+    "Cristina",
+    "Danica",
+    "Diana",
+    "Diane",
+    "Dianne",
+    "Donna",
+    "Elaine",
+    "Ella",
+    "Erica",
+    "Erika",
+    "Grace",
+    "Hannah",
+    "Hazel",
+    "Irene",
+    "Irish",
+    "Ivy",
+    "Jade",
+    "Jane",
+    "Janine",
+    "Jasmine",
+    "Jennifer",
+    "Jenny",
+    "Jessa",
+    "Jessica",
+    "Joanna",
+    "Joy",
+    "Joyce",
+    "Julia",
+    "Justine",
+    "Karen",
+    "Kate",
+    "Katherine",
+    "Kathleen",
+    "Katrina",
+    "Kaye",
+    "Kim",
+    "Kimberly",
+    "Kristine",
+    "Laura",
+    "Lovely",
+    "Mae",
+    "Maria",
+    "Maricar",
+    "Marie",
+    "Mariel",
+    "Marjorie",
+    "Marta",
+    "Mary",
+    "Mary-Ann",
+    "Mary-Grace",
+    "Mary-Joy",
+    "Mary-Rose",
+    "May",
+    "Mia",
+    "Michelle",
+    "Monica",
+    "Nicole",
+    "Nikki",
+    "Nina",
+    "Patricia",
+    "Paula",
+    "Pauline",
+    "Princess",
+    "Queen",
```

Comment:
Are these titles or proper Filipino first names? I imagine to an english-speaking audience (or one not familiar with Filipino culture) these might seem a bit weird as a character is known only as "Princess Sarmiento" or "Queen Dimaculangan" (for example).

Reply:
I just went with the most common names that popped up and didn't censor anything that would sound funny. Not my culture, I know how funny Swedish sounds if you're not from here. Cj is the most far out for me.

> Princess Sarmiento"

https://www.google.com/search?client=ubuntu-sn&channel=fs&q=Princess+Sarmiento

Queen is on the list but doesn't show up as easily as Princess. I can remove Princess or change it for Reyna.
My main method was on testing it in the game and searching the net to see if real persons would pop up.

Reply:
Seems they are indeed actual first names - that clears it up, thanks!